### PR TITLE
Use SolverTools log and released versions

### PR DESCRIPTION
**Pull request by abelsiqueira**

### CODE REVIEW

File: src/trunkls.jl [hunk 1]
```diff
@@ -180,6 +164,10 @@ function trunk(nlp :: AbstractNLSModel;
       end
     end
 
+    @info log_row([iter, f, ∇fNorm2, get_property(tr, :radius), sNorm,
+                   get_property(tr, :ratio), length(cg_stats.residuals), 0, cg_stats.status])
```
Comment (dpo) [inline src/trunkls.jl]: It seems `bk` got lost.

**Comment by abelsiqueira** [comment]
Fixed

**Comment by dpo** [comment]
Thanks. The output looks great! I'm just wondering why the alignment of `cgstatus` is a bit off: https://travis-ci.org/JuliaSmoothOptimizers/JSOSolvers.jl/jobs/507598750#L877 (I would expect `cgstatus` to be left-aligned?!)

**Comment by abelsiqueira** [comment]
The format string uses 15 characters and is right-aligned (`%15s`). Since the string is larger, the formatting is lost. Currently, there is no way to override the formatting.

**Comment by dpo** [comment]
Would it be resolved if all the formats were left-aligned by default (`%-15s`)?

**Comment by abelsiqueira** [comment]
It looks like yes, as long as the status is always the last column.

**Comment by dpo** [comment]
Should that be done in SolverTools?

**Comment by abelsiqueira** [comment]
https://github.com/JuliaSmoothOptimizers/SolverTools.jl/pull/92

**Pull request merged**